Profesja/Include/Player.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <vector>
+
+class Player
+{
+public:
+  uint currentHp = 20;

```

Comment:
20 is a little bit magic number

Czemu ma być default value? Nie lepiej w c-torze?